### PR TITLE
New aws ec2 python script

### LINKED ISSUE #847: Create AWS_EC2-Instance[Python]
# Description

Deploying EC2 instance on AWS using a single python script. I can see there is a Golang script for the same but not the python script.

### PR DESCRIPTION
**Pull request by sakar97**
Fix #847 

This python script will generate a keypair for your aws ec2 instance and also store it in your local machine.
After keypair is generated, it will create an instance provided with the ami-ID.

### CODE REVIEW

**Comment by sakar97** [comment]
@vybhav72954  Please Review my code. Few sanity errors were there which doesn't make sense. Thanks

File: Python/Aws_EC2/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Before we can get started, you'll need to install Boto3 library in Python and the AWS Command Line Interface (CLI) tool using 'pip' which is a package management system written in Python used to install and manage packages that can contain code libraries and dependent files. 
+
+Boto3 is the AWS SDK for Python, which provides Object-based APIs and low-level direct access to AWS services like EC2. AWS CLI is a command line tool written in Python that introduces efficient use cases to manage AWS services with a set of very simple commands.
+
+Using 'pip' run the following command to install the AWS CLI and Python's Boto3 library on your machine:
+
+`pip install awscli boto3`
+
+# AWS Configure to Config AWS Creds
```
Comment (vybhav72954) [inline Python/Aws_EC2/README.md:11]: Use Multiple levels of headings sparingly to make the README more readable.

File: Python/Aws_EC2/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+After creating the user and obtaining the credentials (Access ID and Secret key), we can now configure our Python scripting environment with this credential in order to manage EC2. Use the AWS CI tool to configure these credentials by running the following command from a Bash terminal: 
+
+`aws configure`
+
+It will prompt you to provide the Access Key ID, Secret Key, Default AWS region, and output format. Once those are provided, credentials are saved in a local file at path ~/.aws/credentials and other configurations like region are stored in ~/.aws/config file as demonstrated in the following example.
```
Comment (vybhav72954) [inline Python/Aws_EC2/README.md:17]: Put path in code blocks as well so that they stand out from the normal text

File: Python/Aws_EC2/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+Before we can jump into how to create EC2 instances, it's important to understand how to create a keypair for EC2 instances, so that they can be accessed later, once the virtual machines are launched programmatically using Python.
+
+# Create a New EC2 Instance
+
+Now that we've configured our credentials, let's test if these credentials work well with AWS CLI tools. To do that, run the following command from a Bash shell: 
+
+`aws ec2 describe-instances`
+
+In the above output you can see the Amazon Machine Image (AMI) ID, which looks like ami-00b6a8a2bd28daf19, this is important information, and is required to create a new instance programmatically using Python.
```
Comment (vybhav72954) [inline Python/Aws_EC2/README.md:29]: Introduce an EOF here.

File: Python/Aws_EC2/create_ec2_instance.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import boto3
+
+
+class ec2_instance:
```
Comment (vybhav72954) [inline Python/Aws_EC2/create_ec2_instance.py:4]: The Class Names, COnstant Names and Function names don't follow the proper naming convention.

These will help you:

[Styling Guide](https://github.com/HarshCasper/Rotten-Scripts/blob/master/STYLE_GUIDELINES.md/#Python)
[Official PEP-8 Guide](https://www.python.org/dev/peps/pep-0008/)

File: Python/Aws_EC2/create_ec2_instance.py [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+            print("Error {0}".format(err))
+        else:
+            print("EC2 Instance created", instances)
+
+
+if __name__ == "__main__":
+    ec2Instance = 'ami-0c7945b4c95c0481c'
+    ob = ec2_instance(ec2Instance)
+    ob.key_pair()
+    ob.create_ec2()
```
Comment (vybhav72954) [inline Python/Aws_EC2/create_ec2_instance.py:46]: Introduce EOF, please.

**Review by vybhav72954 — CHANGES_REQUESTED** [review]
Hey, @sakar97 the PR looks good. Although I am yet tO test it on my machine, thus can't validate implementation, (I will do it tomorrow).

I have added a few other minor changes, please get them done soon.

File: Python/Aws_EC2/create_ec2_instance.py [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+                KeyName='ec42-keypair'
+            )
+        except Exception as err:
+            print("Error {0}".format(err))
+        else:
+            print("EC2 Instance created", instances)
+
+
+if __name__ == "__main__":
+    ec2Instance = 'ami-0c7945b4c95c0481c'
```
Comment (HarshCasper) [inline Python/Aws_EC2/create_ec2_instance.py:43]: Why does this look like a secret? Also can you use proper variable names?

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: Python/Aws_EC2/create_ec2_instance.py (hunk 6 above)
Comment (sakar97) [reply Python/Aws_EC2/create_ec2_instance.py:43]: instance-id is unique to every user and for every region so it is not required to make it encrypted.

Comment (HarshCasper) [inline Python/Aws_EC2/create_ec2_instance.py:43]: If its unique, can you keep them in a `.env` file and use a module like decouple to import them?

**Review by HarshCasper — CHANGES_REQUESTED** [review]
One tiny change: Rest looks good to merge

File: Python/Aws_EC2/create_ec2_instance.py (hunk 6 above)
Comment (vybhav72954) [inline Python/Aws_EC2/create_ec2_instance.py:43]: @sanket143 Something like this

Please don't upload the .env file and in its place upload .env.example

Comment (sakar97) [reply Python/Aws_EC2/create_ec2_instance.py:43]: yes, understood! will update in 5 minutes.

**Review by vybhav72954 — APPROVED** [review]
LGTM

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**